tests/integration_tests/tests/test_cases.py

```diff
@@ -102,6 +102,7 @@ def tearDown(self):
     def _save_manager_logs_after_test(self, purge=True):
         self.logger.debug('_save_manager_logs_after_test started')
         logs_dir = os.environ.get('CFY_LOGS_PATH')
+        self.logger.info("CFY_LOGS_PATH: {}".format(logs_dir))
```

Comment:
Add more description, mention that this is the remote and you can add the local (if it's set).

Reply:
:+1: